src/plugins/workspace/public/components/workspace_use_case_overview_app.tsx

```diff
@@ -31,9 +49,119 @@ export const WorkspaceUseCaseOverviewApp = (props: WorkspaceUseCaseOverviewProps
     chrome.setBreadcrumbs(breadcrumbs);
   }, [chrome, currentWorkspace]);
 
-  const pageId = props.pageId;
+  const { pageId } = props;
+
+  const uiSettingsKeyMap: Record<string, string> = {
```

Comment:
I am thinking if we can make the dismiss button as a reuseable component, with the settingKey / text as its props.

Reply:
that's good idea, need find out a shared place for this component.